bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="omnilink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- OmniLink Controller Bridge -->
+	<bridge-type id="controller">
+		<label>OmniLink Controller</label>
+		<description>An OmniLink controller.</description>
+		<channels>
+			<channel id="sysdate" typeId="sysDate"/>
+			<channel id="enable_disable_beeper" typeId="console_enable_disable_beeper">
+				<label>Enable/Disable All Console Beepers</label>
+			</channel>
+			<channel id="beep" typeId="console_beep">
+				<label>Beep All Consoles</label>
+			</channel>
+			<channel id="last_log" typeId="last_log"/>
+			<channel id="phone_line_event" typeId="phone_line_event"/>
+			<channel id="ac_power_event" typeId="ac_power_event"/>
+			<channel id="battery_event" typeId="battery_event"/>
+			<channel id="dcm_event" typeId="dcm_event"/>
+			<channel id="energy_cost_event" typeId="energy_cost_event"/>
+			<channel id="camera_trigger_event" typeId="camera_trigger_event"/>
+			<channel id="upb_link_activated_event" typeId="upb_link_activated_event"/>
+			<channel id="upb_link_deactivated_event" typeId="upb_link_deactivated_event"/>
+		</channels>
+		<properties>
+			<property name="model number"/>
+			<property name="major version"/>
+			<property name="minor version"/>
+			<property name="revision"/>
+			<property name="phone number"/>
+		</properties>
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP or Host Name</label>
+				<description>The IP or host name of the controller.</description>
+			</parameter>
+			<parameter name="port" type="integer" required="true">
+				<context>network-port</context>
```

Comment:
Are you sure this context exists?

Reply:
What do you mean?

Reply:
`network-port` is not documented here https://www.openhab.org/docs/developer/bindings/config-xml.html#xml-structure-for-configuration-descriptions

I was wondering if you found an undocumented feature.

Reply:
I actually don't know. It has been that way since i started developing for this binding. @digitaldan @craigham any idea?

Reply:
Not sure.  I believe @digitaldan may have written this.   It is just the ip port specification, however other approved bindings do this should be used.

Reply:
who knows, this was written long ago when we were moving from 1.x to 2.x and was probably a misunderstanding of the new XML format at the time. If you run into this type of thing its obviously just best to fix it since this clearly violates  the schema definition.

Reply:
I'll remove it then